.jenkins/actions/run_standalone.sh

```diff
@@ -0,0 +1,112 @@
+if [ "${SAVE_CACHE}" != "true" -a "${DO_PROFILE}" != "true" ] ; then
+    SAVE_TIMINGS="true"
+fi
+
+# configuration
+SCRIPT=`realpath $0`
+SCRIPTPATH=`dirname $SCRIPT`
+ROOT_DIR="$(dirname "$(dirname "$SCRIPTPATH")")"
+FORTRAN_SERIALIZED_DATA_VERSION=7.2.5
+TIMESTEPS=2
+RANKS=6
+BENCHMARK_DIR=${ROOT_DIR}/examples/standalone/benchmarks
+DATA_DIR="/project/s1053/fv3core_serialized_test_data/${FORTRAN_SERIALIZED_DATA_VERSION}/${experiment}"
+ARTIFACT_DIR="/project/s1053/performance/fv3core_monitor/${backend}"
+CACHE_DIR="/scratch/snx3000/olifu/jenkins/scratch/store_gt_caches/${experiment}/${backend}"
+
+# check sanity of environment
+test -n "${experiment}" || exitError 1001 ${LINENO} "experiment is not defined"
+test -n "${backend}" || exitError 1002 ${LINENO} "backend is not defined"
+if [ ! -d "${DATA_DIR}" ] ; then
+    exitError 1003 ${LINENO} "test data directory ${DATA_DIR} does not exist"
+fi
+if [ ! -d "${ARTIFACT_DIR}" ] ; then
+    exitError 1004 ${LINENO} "Artifact directory ${ARTIFACT_DIR} does not exist"
+fi
+if [ ! -d "${BENCHMARK_DIR}" ] ; then
+    exitError 1005 ${LINENO} "Benchmark directory ${BENCHMARK_DIR} does not exist"
+fi
+
+# echo config
+echo "=== $0 configuration ==========================="
+echo "Script:               ${SCRIPT}"
+echo "Do profiling:         ${DO_PROFILE}"
+echo "Save GT4Py caches:    ${SAVE_CACHE}"
+echo "Save timings:         ${SAVE_TIMINGS}"
+echo "Root directory:       ${ROOT_DIR}"
+echo "Experiment:           ${experiment}"
+echo "Backend:              ${backend}"
+echo "Fortran data version: ${FORTRAN_SERIALIZED_DATA_VERSION}"
+echo "Timesteps:            ${TIMESTEPS}"
+echo "Ranks:                ${RANKS}"
+echo "Benchmark directory:  ${BENCHMARK_DIR}"
+echo "Data directory:       ${DATA_DIR}"
+echo "Artifact directory:   ${ARTIFACT_DIR}"
+echo "Cache directory:      ${CACHE_DIR}"
+
+# run standalone
+echo "=== Running standalone ========================="
+if [ "${DO_PROFILE}" == "true" ] ; then
+    profile="--profile"
+fi
+cmd="${BENCHMARK_DIR}/run_on_daint.sh ${TIMESTEPS} ${RANKS} ${backend} ${DATA_DIR} '' '${profile}'"
+echo "Run command: ${cmd}"
+${cmd}
+
+echo "=== Post-processing ============================"
+
+# store timing artifacts
+if [ "${SAVE_TIMINGS}" == "true" ] ; then
+    echo "Copying timing information to ${ARTIFACT_DIR}"
+    cp $ROOT_DIR/*.json ${ARTIFACT_DIR}/
+fi
+
+# store cache artifacts (and remove caches afterwards)
+if [ "${SAVE_CACHE}" == "true" ] ; then
+    echo "Copying GT4Py cache directories to ${CACHE_DIR}"
+    mkdir -p ${CACHE_DIR}
+    rm -rf ${CACHE_DIR}/.gt_cache*
+    cp -rp .gt_cache* ${CACHE_DIR}/
+fi
+rm -rf .gt_cache*
+
+# run analysis and store profiling artifacts
+if [ "${DO_PROFILE}" == "true" ] ; then
+    echo "Analyzing profiling results"
+    ${BENCHMARK_DIR}/process_profiling.sh
+    echo "Copying profiling information to ${ARTIFACT_DIR}"
+    cp $ROOT_DIR/*.prof ${ARTIFACT_DIR}/
+fi
+
+# remove venv (too many files!)
+rm -rf $ROOT_DIR/venv
```

Comment:
I am sure it doesn't matter at the end here, but might do a 'deactivate' first?

Reply:
There is no `source .../activate` in this script, so not needed.

Reply:
looks like nothing that activates the venv does a deactivate as a final step, so it probably still is activated at this point. I guess it doesn't matter.

Reply:
It is built and activated inside `run_on_daint.sh`. But once you exit a shell script venv's are deactivated. This is really just to make sure it is removed.

Reply:
ah didn't realize that, makes sense